src/macOSNotif.js

```diff
@@ -35,100 +36,100 @@ class __macOSNotifJS_Interact {
+            this.dragXOffset = evt.clientX;
         } else if (evt.type === "touchstart") {
-            this.drag_xOffset = evt.targetTouches[0].clientX;
+            this.dragXOffset = evt.targetTouches[0].clientX;
         }
 
-        if (this.drag_xOrg === null) this.drag_xOrg = this.drag_rightOffset();
-        this.drag_acting = true;
+        if (this.dragXOrg === null) this.dragXOrg = this.dragRightOffset();
+        this.dragActing = true;
     }
 
-    drag_stop(evt) {
-        if (!this.drag_acting) return;
+    dragStop(evt) {
+        if (!this.dragActing) return;
         evt.preventDefault();
         evt.stopPropagation();
 
         // Reset transition and stop dragging
         this.element.style.transition = "";
-        this.drag_acting = false;
+        this.dragActing = false;
 
         // Check if we should dismiss
-        const offset = Math.abs(this.drag_rightOffset());
+        const offset = Math.abs(this.dragRightOffset());
         const threshold = this.element.offsetWidth * 0.2;
         if (offset >= threshold) {
             this.onDismiss();
         } else {
-            this.element.style.right = this.drag_xOrg + "px";
+            this.element.style.right = this.dragXOrg + "px";
         }
     }
 
-    drag_run() {
-        this.element.addEventListener("mousedown", (evt) => this.drag_start(evt), true);
-        this.element.addEventListener("touchstart", (evt) => this.drag_start(evt), true);
+    dragRun() {
+        this.element.addEventListener("mousedown", (evt) => this.dragStart(evt), true);
+        this.element.addEventListener("touchstart", (evt) => this.dragStart(evt), true);
 
-        window.addEventListener("mousemove", (evt) => this.drag_move(evt), true);
-        window.addEventListener("touchmove", (evt) => this.drag_move(evt), true);
+        window.addEventListener("mousemove", (evt) => this.dragMove(evt), true);
+        window.addEventListener("touchmove", (evt) => this.dragMove(evt), true);
 
-        window.addEventListener("mouseup", (evt) => this.drag_stop(evt), true);
-        window.addEventListener("touchend", (evt) => this.drag_stop(evt), true);
+        window.addEventListener("mouseup", (evt) => this.dragStop(evt), true);
+        window.addEventListener("touchend", (evt) => this.dragStop(evt), true);
     }
 
     run() {
-        this.drag_run();
+        this.dragRun();
 
         // TODO: this
-        //this.scroll_run();
+        //this.scrollRun();
     }
 }
 
-const __macOSNotifJS_template = (require("./html/macOSNotif.html").default).replace(/<!--(?!>)[\S\s]*?-->/g, ""); // Strip HTML comments
-const __macOSNotifJS_notifs = {};
-let __macOSNotifJS_fadeThreshold = 6;
-const __maOSNotifJS_themes = {
+const __macOSNotifJSTemplate = (require("./html/macOSNotif.html").default).replace(/<!--(?!>)[\S\s]*?-->/g, ""); // Strip HTML comments
+const __macOSNotifJSNotifs = {};
+let __macOSNotifJSFadeThreshold = 6;
+const __maOSNotifJSThemes = {
```

Comment:
We could store these as class variables instead of variables here?

Reply:
Private static class variables don't exist in JS, so this isn't possible for now.